### PR TITLE
Introduces item/entry/record content

### PR DESCRIPTION
**Pull request by subatoi**
Introduces this content and a worked example

### CODE REVIEW

**Review by arnau — DISMISSED** [review]

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (danmitchell-) [inline source/documentation/using_the_api/item_entry_record_example.md]: In the frontend we use the phrase `update` instead of `change`

**Review by dankmitchell — APPROVED** [review]

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (bahmady) [reply source/documentation/using_the_api/item_entry_record_example.md]: Good spot @danmitchell- and thanks for the review. I'll change this now.

Just FYI I'm waiting on approval from the tech writing team before I can publish this.

**Review by dankmitchell — DISMISSED** [review]

**Review by michaelabenyohai — DISMISSED** [review]

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (danmitchell-) [inline source/documentation/using_the_api/item_entry_record_example.md]: Would we be better off starting with the `entry`?

**Review by dankmitchell — COMMENTED** [review]
Just a question

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: > the `official-name` field for this entry

It's really the field for the *item* rather than the entry. In another paragraph we use the below, which seems a bit inconsistent:

> the `official-name` field for the corresponding item

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: We have lost that detail that specifies that the record also contains the item data already resolved (whereas an entry does not). Was this intentional?

Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: >  'GET /records/CI`

This doesn't seem to be formatted correctly.

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (nacnudus) [inline source/documentation/using_the_api/item_entry_record_example.md]: I think some more links would help.  For example, a link to the [corresponding item](https://country.register.gov.uk/items/sha-256:7c16257bd45b4716914010b39dd40e5a6b985b8928d7b8bb0fe3005d2f2b0fec.json) of the 90th entry would clarify that `sha-256:` is part of the item hash (I just made this mistake).

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: > For example, `GET /records/CI` returns the 207th entry.

I wonder whether it's worth adding "and its corresponding item". This is because this is the same description used for `GET /records/CI/entries` below, but the endpoints return quite different things (the former returns the entry and item, the latter only the entries).

> Using the `GET /records/CI/entries` endpoint will return both the 90th and the 207th entries.

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (nacnudus) [inline source/documentation/using_the_api/item_entry_record_example.md]: [corresponding item](https://country.register.gov.uk/items/sha-256:b3ca21b3b3a795ab9cd1d10f3d447947328406984f8a461b43d9b74b58cccfe8.json)

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (nacnudus) [inline source/documentation/using_the_api/item_entry_record_example.md]: [most recent entry](https://country.register.gov.uk/records/CI.json)

Comment (nacnudus) [reply source/documentation/using_the_api/item_entry_record_example.md]: Also, `most recent entry to the most register`?

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (nacnudus) [inline source/documentation/using_the_api/item_entry_record_example.md]: [all entries for the key 'CI'](https://country.register.gov.uk/records/CI/entries.json)

**Review by nacnudus — CHANGES_REQUESTED** [review]
Just suggesting a few more links, and a possible typo.

**Review by nacnudus — APPROVED** [review]

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: Can we change the first to the second:

> returns the 207th entry and [its corresponding item]( https://country.register.gov.uk/records/CI/.json)

> returns the [207th entry and its corresponding item]( https://country.register.gov.uk/records/CI.json)

Include "207th entry" in the linked text (as this is part of the response) and lose the erroneous `/` from the URL.

File: source/documentation/using_the_api/item_entry_record_example.md
Comment (michaelabenyohai) [inline source/documentation/using_the_api/item_entry_record_example.md]: > https://country.register.gov.uk/items/sha-256:b3ca21b3b3a795ab9cd1d10f3d447947328406984f8a461b43d9b74b58cccfe8

needs `.json` on the end of it

> https://country.register.gov.uk/items/sha-256:b3ca21b3b3a795ab9cd1d10f3d447947328406984f8a461b43d9b74b58cccfe8.json

**Review by michaelabenyohai — CHANGES_REQUESTED** [review]
Just a couple of small things above

**Review by michaelabenyohai — APPROVED** [review]
💯

**Pull request merged**
